jupyterlab/labapp.py

```diff
@@ -15,6 +16,25 @@ class LabApp(NotebookApp):
         help="The default URL to redirect to from `/`"
     )
 
+    extra_lab_extensions_path = List(Unicode(), config=True,
+        help="""extra paths to look for JupyterLab extensions"""
+    )
+
+    lab_extensions = Dict({}, config=True,
+        help=("Dict of Python modules to load as notebook server extensions."
+              "Entry values can be used to enable and disable the loading of"
+              "the extensions.")
+    )
+
+    @property
+    def lab_extensions_path(self):
+        """The path to look for JupyterLab extensions"""
+        return self.extra_lab_extensions_path + jupyter_path('lab_extensions')
```

Comment:
both nbextensions and serverextensions don't have an underscore, so for consistency we should probably call this `labextensions`. Does that look too terrible?